docs/modules/ROOT/examples/formatExamples.cmd

```diff
@@ -0,0 +1 @@
+jbang run jbang-fmt@jbangdev/jbang-fmt --line-length 90 --java 21 --indent-with space --indent-size 2 src
```

Comment:
instead of having 2 files for these just make a jbang-catalog file with a jbang-fmt alias in ...then you just need to run `jbang jbang-fmt` no matter what OIS you are on

Reply:
btw. why not just include this source folder in the spotlessapply build? then its formatted automatically?